spec/services/sufia/workflow/changes_required_notification_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+RSpec.describe Sufia::Workflow::ChangesRequiredNotification do
+  let(:depositor) { create(:user) }
+  let(:cc_user) { create(:user) }
+  let(:work) { create(:generic_work, user: depositor) }
+  let(:entity) { create(:sipity_entity, proxy_for_global_id: work.to_global_id.to_s) }
+  let(:comment) { double("comment", comment: 'A pleasant read') }
+  let(:recipients) { { 'to' => [depositor], 'cc' => [cc_user] } }
```

Comment:
The `to` user is the one with the role `approving`, per the above workflow, not the depositor.